autoload/vimlparser.vim

```diff
@@ -3914,51 +3919,51 @@ function! s:Compiler.compile(node)
   if a:node.type == s:NODE_TOPLEVEL
     return self.compile_toplevel(a:node)
   elseif a:node.type == s:NODE_COMMENT
-    return self.compile_comment(a:node)
+    call self.compile_comment(a:node)
```

Comment:
I wonder this change may go wrong. Vim-script  parse each lines even though the lines are not passing. So early return makes good performance result.